WatchFaceAlphaKotlin/app/build.gradle

```diff
@@ -61,6 +65,14 @@ dependencies {
     api "org.jetbrains.kotlinx:kotlinx-coroutines-core:$rootProject.coroutines"
     api "org.jetbrains.kotlinx:kotlinx-coroutines-android:$rootProject.coroutines"
 
+    implementation "androidx.activity:activity-ktx:1.3.1"
+
+    // Lifecycle components
+    implementation "androidx.lifecycle:lifecycle-runtime-ktx:2.3.1"
+    implementation "androidx.lifecycle:lifecycle-viewmodel-ktx:2.3.1"
+
+    // Material components
+    api 'com.google.android.material:material:1.4.0'
```

Comment:
Small nit, but is there a reason that these are `api`? Unless another module is depending on this one, `api` doesn't really mean anything over `implementation`.